isamples_metadata/OpenContextTransformer.py

```diff
@@ -0,0 +1,217 @@
+        citation_uri = self._citation_uri()
+        return f"metadata/{citation_uri.removeprefix(self.N2T_ARK_PREFIX)}"
+
+    def sample_identifier_scheme(self) -> typing.AnyStr:
+        pass
+
+    def sample_identifier_string(self) -> typing.AnyStr:
+        return self._citation_uri().removeprefix(self.N2T_PREFIX)
+
+    def sample_identifier_value(self) -> typing.AnyStr:
+        # TODO is this even used?
+        return self._citation_uri().removeprefix(self.N2T_PREFIX)
+
+    def sample_label(self) -> typing.AnyStr:
+        return self.source_record.get("label", Transformer.NOT_PROVIDED)
+
+    def sample_description(self) -> typing.AnyStr:
+        description_pieces = []
+        self._transform_key_to_label(
+            "early bce/ce", self.source_record, description_pieces
+        )
+        self._transform_key_to_label(
+            "late bce/ce", self.source_record, description_pieces
+        )
+        self._transform_key_to_label("updated", self.source_record, description_pieces)
+        self._transform_key_to_label("updated", self.source_record, description_pieces)
+        for consistsOfDict in self.source_record.get("Consists of", []):
+            self._transform_key_to_label(
+                "label", consistsOfDict, description_pieces, "Consists of"
+            )
+        for hasTypeDict in self.source_record.get("Has type", []):
+            self._transform_key_to_label(
+                "label", hasTypeDict, description_pieces, "Has type"
+            )
+        return Transformer.DESCRIPTION_SEPARATOR.join(description_pieces)
+
+    def sample_registrant(self) -> typing.AnyStr:
+        pass
+
+    def sample_sampling_purpose(self) -> typing.AnyStr:
+        pass
+
+    def has_context_categories(self) -> typing.List[typing.AnyStr]:
+        return ["Site of past human activities"]
+
+    def has_material_categories(self) -> typing.List[typing.AnyStr]:
+        item_category = self.source_record.get("item category")
+        return MaterialCategoryMetaMapper.categories(item_category)
+
+    def has_specimen_categories(self) -> typing.List[typing.AnyStr]:
+        item_category = self.source_record.get("item category")
+        return SpecimenCategoryMetaMapper.categories(item_category)
+
+    def _context_label_pieces(self) -> typing.List[typing.AnyStr]:
+        context_label = self.source_record.get("context label")
+        if len(context_label) > 0:
+            return context_label.split("/")
+        else:
+            return []
+
+    def keywords(self) -> typing.List[typing.AnyStr]:
+        return self._context_label_pieces()
+
+    def produced_by_id_string(self) -> typing.AnyStr:
+        return Transformer.NOT_PROVIDED
+
+    def produced_by_label(self) -> typing.AnyStr:
+        return self.source_record.get("project label", Transformer.NOT_PROVIDED)
+
+    def produced_by_description(self) -> typing.AnyStr:
+        return self.source_record.get("project uri", Transformer.NOT_PROVIDED)
```

Comment:
Eric Kansa -- the textual description of this one isn't included in the version of the record we pull down, so we are substituting with the project uri.  If we want the textual description here, e.g. "Objects from the Short Barracks of Fort Snelling, at the confluence of the Mississippi and Minnesota Rivers", instead of https://opencontext.org/projects/fab0532a-2953-4f13-aa97-8a9d7e992dbe